### DISCUSSION TITLE
Why own SQL parser was used instead of the ready solutions?

### DISCUSSION THREAD

**Discussion by ice-myles** [asdine/genji#522] [General]
Why own SQL parser was written instead of the ready solutions like [vitess-sqlparser](https://github.com/blastrain/vitess-sqlparser) or [xwb1989/sqlparser](https://github.com/xwb1989/sqlparser), any else?

**Comment by samjewell** [comment]
Maybe because those SQL parsers you reference are for the MySQL dialect, while this project aims to be compatible with the Postgres dialect?
